src/resmom/mom_main.c

```diff
@@ -7959,42 +7963,23 @@ int
 main(int argc, char *argv[])
 #endif
 {
-	static	char		id[] = "mom_main";
-	struct			tpp_config tpp_conf;
-	int	 		errflg, c, rc;
-	int			stalone = 0;
-	int			i;
-#ifndef	DEBUG
-	FILE			*dummyfile;
-#endif
-#ifdef	WIN32
-	struct arg_param	*p = (struct arg_param *)pv;
-	int			argc;
-	char		 	**argv;
-	SERVICE_STATUS		ss;
-	int			pmode = S_IREAD | S_IWRITE;
-#else
-	int			pmode = S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH;
-
-	struct sigaction	act;
-#endif /* WIN32 */
+	/* both Win32 and Unix */
+	struct tpp_config	tpp_conf;
```

Comment:
Is the indentation off here?  It looks like it lined up before and doesn't any more.  Could you check the indentation for all the variables?